src/components/PrimaryPanes/OutlineFilter.js

```diff
@@ -35,6 +35,8 @@ export default class OutlineFilter extends Component<Props, State> {
       // also bound to the ESC key
       e.preventDefault();
       this.props.updateFilter("");
+    } else if (e.key === "Enter") {
+      e.preventDefault();
```

Comment:
could we add a comment explaining why it's needed ? With maybe a reference to the issue.